src/avfoundation.cs

```diff
@@ -3844,6 +3950,18 @@ interface AVAssetResourceLoadingContentInformationRequest {
 		string[] AllowedContentTypes { get; }
 	}
 
+	interface IAVAssetWriterDelegate { }
+
+	[TV (14,0), NoWatch, Mac (11,0), iOS (14,0)]
+	[Protocol]
```

Comment:
This should be a Model too:

```suggestion
	[Protocol]
	[Model (AutoGeneratedName = true)]
	[BaseType (typeof (NSObject))]
```